### ISSUE TITLE
Shared Hosting Firewall warnings

### ISSUE THREAD

**Issue by notbasetwo** [FOSSBilling/fossbilling.org#4]
Quite a lot of the issues I have seen surrounding connecting to servers in hosting modules or doing WHOIS checks are likely down to the end-user's local configuration.

I suspect a lot of them are due to over zealous firewalls implemented by shared hosts (blocking non-standard outbound ports, or more likely just using the CSF defaults).

When writing the documentation for shared host installation/documentation for integrations - it could be worth mentioning to try and avoid those kinds of issues being raised when it's nothing to do with the codebase.

**Comment by John-S4** [comment]
Yep, I'll add something about the ports in the installation (or maybe after installation) section.

**Comment by admdly** [comment]
Firewall guidance is covered in `security/best-practices`.
